### PR TITLE
Refactor DesignSystem, markup and css

### LINKED ISSUE #115: Unify design system, css and markup
Collects all refactoring around the `DesignSystem` directory and styling in general. 

**Guidelines/README**
- [x] Define what belongs into the `DesignSystem` directory 
- [x] Define what belongs onto the `DesignSystem` page
- [x] Create guidelines for the use of styling

**Consistent design system components**
- [x] move `Numeric`, `Text`, `Title` each into a single component
- [x] Rename Inputs 
- [x] move input validation errors into Input component
- [x] possibly extract any other reusable components 
  - [x]  ~~`Row`~~  _postponed to #117 _
  - [x]  `button-row`
  - [x]  ~~`StepCounter`~~  _postponed to #117 _
  - [x]  `Avatar`/`Title` combination - see eg `PickWalletStep` 

**Reduce inline css**
- [x] ~~move inline css of components into params or classes~~ _There is currently no good way to pass classes and styling to components in a different way. We'll get back to it when an alternative is implemented._
- [x] ~~Can some of the inline styling be moved in to a parent `div`?~~ _We decided that we want to avoid wrapping components into div's for minor styling changes, as it will clutter the code._
- [x] remove all inline css of non-components (`FileList`, `TransactionSummaryStep`, `PickWalletStep`, `PickNameStep`,  `ConfirmTransactionStep`, `CreateProject`, `RegisterProject`)

**Consistent css**
- [x] ~~reduce duplication of css (`Button`, global styling)?~~
- [x] restructure class names consistently according to hierarchy: layout/page > container > content
- [x] ~use explicit class names for `style` if it is specific to a component~

**Consistent markup**
- [x] remove duplicate wrapping of components in div's (eg `Project`) 

Refs #97

### PR DESCRIPTION
**Pull request by MeBrei**
Resolves #115

### CODE REVIEW

File: app/src/DesignSystem/Numeric.svelte [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<script>
+  export let style = null;
+  export let size = null;
+
+  let sizeClass = size ? size : "regular";
+</script>
+
+<style>
+  small {
```
Comment (rudolfs) [inline app/src/DesignSystem/Numeric.svelte]: Typo?

Comment (MeBrei) [reply app/src/DesignSystem/Numeric.svelte]: Yes! Thanks for pointing it out

**Comment by rudolfs** [comment]
Just noticed this when starting up the app:
```
(!) Plugin svelte: ButtonRow has unused export property 'style'. If it is for external reference only, please consider using `export const style`
src/DesignSystem/ButtonRow.svelte
1: <script>
2:   export let style = null;
                ^
3: </script>
```

File: app/src/DesignSystem/Text.svelte [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<script>
+  export let style = null;
+  export let size = null;
+  export let weight = null;
+
+  // let sizeClass = size === "small" ? "smallSize" : "regularSize";
+  // let weightClass = size === "bold" ? "boldWeight" : "regularWeight";
```
[the remark this answers is not in the document]
Comment (MeBrei) [reply app/src/DesignSystem/Text.svelte]: Removed!

File: app/src/DesignSystem/ButtonRow.svelte [hunk 3]
```diff
@@ -0,0 +1,26 @@
+<script>
```
[the remark this answers is not in the document]
Comment (MeBrei) [reply app/src/DesignSystem/ButtonRow.svelte]: Sounds good!

**Pull request merged**